src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c

```diff
@@ -485,7 +485,7 @@ void func_809289E4(ObjTsubo* this, PlayState* play) {
         func_80927818(this, play, 1);
         SoundSource_PlaySfxAtFixedWorldPos(play, &this->actor.world.pos, 20, NA_SE_EV_POT_BROKEN);
         if (ObjTsubo_IsSceneNotGohtOrTwinmold(this, play)) {
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```

Comment:
Can replace with an early return.

Reply:
Does not match:
![image](https://user-images.githubusercontent.com/12245827/195779298-598ded2f-83d0-47e1-8570-2dc27d458cf8.png)